src/bootupd.rs

```diff
@@ -58,10 +58,11 @@ struct StatusOptions {
     json: bool,
 }
 
+// Options exposed by `bootupctl backend`
```

Comment:
Nit: proper docstrings need three slashes, `///`.

Reply:
I think that would result in structopt showing them in `--help` which I don't actually want here.